### DISCUSSION TITLE
Where are we on the Diffusions of Innovation?

### DISCUSSION THREAD

**Discussion by MichaelRimler** [rinpharma/positconf-roundtables-2023#8] [Ideas]
Many of the other ideas in this discussion are quite technically related. Even the 'adoption' seemed to be related to companies' buy-in, not the user buy-in. Where are we on getting data analysts and data scientists that work with clinical data on board (in particular, those delivering CSRs and submission packages)? What are the challenges - what has been overcome - and let's be sure to acknowledge that the companies represented in these round-tables are further along their adoption journey than many others that are attending to learn.

**Comment by MichaelRimler** [comment]
Law of Diffusion of Innovations, explained by Simon Sinek:
https://youtu.be/-g_FK1qEIZQ

Wikipedia: https://en.wikipedia.org/wiki/Diffusion_of_innovations

Easy to read article: https://crm.org/articles/diffusion-of-innovations

**Comment by minhonglee** [comment]
I think this dives into the whole change management element of adopting open source.  It would be good to talk about the various aspects of change that must be managed, including at the user (data scientist, analyst, programmer) level.  I think there are a number of challenges here.  In my experience, there's a general excitement and interest but it's not necessarily focused most of the time.  They want to learn and use it but not sure for what purpose or how.  I also don't think they realize that there's a significant learning curve.  It would be great to talk about this as I'm sure different companies could have different experiences with this.    

I also see this as just one of the change management issues as there are others in regards to changing from one platform to either adopting or incorporating another.  Different organizations/teams are impacted differently and have different views of adoption.  Some are more top down and others are bottom up in regards to driving adoption.  I think there are again many challenges here and wonder how others have dealt with it.

**Reply by Coutuan3s** [reply]
At Sanofi we are still dealing with it... There's the submission ready env (OS, R version, Packages, IDE, ...), the people upskilling in R (multi year initiative), then finding work to ensure knowledge is applied, the process changes (package management, ...), the new tools (RStudio, Git, internal packages...) with relevant trainings, the new way of doing your custom work... There is a need for onboarding user champions to help other in their path to change, there is new collaborations with IT to set the different assets, the leadership commitment, messages, and setting targets, the business value and ROI assessments to support budget needs, and more ...

**Reply by mattkumar** [reply]
Great comments @minhonglee. There's so much to unpack in the learning curve component. I think @MichaelRimler nailed it (paraphrasing) in highlighting the differences between traditional statistical programming and software engineering/developer mindsets, and how elements of each are necessary to use not only to utilize new tools and workflows to complete tasks, but also to collaborate across industry.  At Bayer, our computational sciences team have been working to introduce all these concepts to statistical programmers at the appropriate time through structured workshops, presentations, and in the near future, job-shadowing. Definitely upvoting this!

**Reply by harveyl888** [reply]
Agree with all the above.  Upskilling and changing mindset (from statistical programmer to software engineer) is huge.  A couple of weeks ago I introduced a SAS-based colleague (who has some R experience) to GitLab and markdown and it blew their mind.  Hard to believe you can get this far in programming without using tools which, to me, are indispensable.
It would also be interesting to hear approaches taken within Biotech where cost may drive more innovative ways of working.

**Reply by MichaelRimler** [reply]
We had similar experiences (not all solved) - which is the push behind this. There's enabling the tech - and enabling the people!
